contribs/gmf/src/directives/locationchooser.js

```diff
@@ -76,7 +73,14 @@ gmf.LocationchooserController = function($scope) {
    * @type {gmfx.LocationchooserLocation}
    * @export
    */
-  this.selectedLocation = this.locations[0];
+  this.selectedLocation;
+
+  if (!goog.isDef(this.selectedLocation)) {
+    this.selectedLocation = this.locations[0];
+  }
+
+  $scope.$watch('ctrl.selectedLocation', goog.bind(this.setLocation, this));
```

Comment:
Did you see my comment suggesting to use a function instead of an expression as the first arg to `$watch`?